### PR TITLE
Query Browser: Add border-radius for series buttons

### PR DESCRIPTION
**Pull request by kyoto**
To match the other buttons on the page.

FYI @cshinn

### CODE REVIEW

File: frontend/public/components/monitoring/_monitoring.scss [hunk 1]
```diff
@@ -106,6 +106,7 @@
 
 .query-browser__series-btn-wrap {
   border: 1px solid transparent;
+  border-radius: 3px;
```
Comment (spadgett) [inline frontend/public/components/monitoring/_monitoring.scss]:
```suggestion
  border-radius: var(--pf-global--BorderRadius--sm);
```

Comment (kyoto) [reply frontend/public/components/monitoring/_monitoring.scss]: Changed. Thanks!

**Pull request merged**